projects/kit/components/avatar/avatar.template.html

```diff
@@ -9,6 +9,6 @@
 <tui-svg
     *ngIf="iconAvatar"
     class="t-icon"
-    [src]="avatarUrl || ''"
+    [src]="$any(avatarUrl || '')"
```

Comment:
We can avoid typecasting if we make a getter:
```ts
get stringAvatar(): string {
  return this.iconAvatar ? this.avatarUrl : '';
}
```
and
```html
<tui-svg
  *ngIf="stringAvatar"
```

Reply:
Great idea! :blush: 

Thanks so much for the input! I also added an "extra" `typeof` check to narrow it down to `string`

Reply:
Typeof check is nested in the getter, so you can probably just cast it to string to shorten it since we only need it for the type:
```ts
return this.iconAvatar ? String(this.avatarUrl) : '';
```
Also there's no need for empty string fallback anymore;
```html
[src]="stringAvatar || ''"
```

Reply:
Good catches! Pushed the suggested changes